ledger/src/get.rs

```diff
@@ -87,6 +87,26 @@ impl<N: Network, C: ConsensusStorage<N>> Ledger<N, C> {
         Ok(epoch_hash)
     }
 
+    /// Returns the provers and the number of solutions they have submitted for the current epoch.
+    pub fn get_epoch_provers(&self) -> Arc<RwLock<IndexMap<Address<N>, u32>>> {
```

Comment:
nit: ideally, I'd return just `RwLock<IndexMap<Address<N>, u32>>` from here, or even just `IndexMap<Address<N>, u32>`, for broader utility